src/css/showHide.js

```diff
@@ -4,6 +4,23 @@ define( [
 	"../css/var/isHidden"
 ], function( jQuery, dataPriv, isHidden ) {
 
+var defaultDisplayMap = {};
+
+function setDefaultDisplay( elem ) {
```

Comment:
To to be clear, without that hook we loosing two use-cases:
1. When `body`/`html` is hidden
2. When all specific node-types are hidden, like 
```css
li {display: none}
```

First might exist in at least worlds like SPA or iframes, there is probably more weird use-cases out.
As of all specific elements being hidden, there might be some legit cases for very simple pages.

I think it would be good to provide some workarounds for it.